src/broadcasts/broadcasts.ts

```diff
@@ -0,0 +1,56 @@
+import { renderAsync } from '@react-email/render';
+import type * as React from 'react';
+import type { Resend } from '../resend';
+import type {
+  CreateBroadcastOptions,
+  CreateBroadcastRequestOptions,
+} from './interfaces/create-broadcast-options.interface';
+import type {
+  SendBroadcastOptions,
+  SendBroadcastResponse,
+  SendBroadcastResponseSuccess,
+} from './interfaces/send-broadcast-options.interface';
+
+export class Broadcasts {
+  constructor(private readonly resend: Resend) {}
+
+  async create(
+    payload: CreateBroadcastOptions,
+    options: CreateBroadcastRequestOptions = {},
+  ): Promise<SendBroadcastResponse> {
+    if (payload.react) {
+      payload.html = await renderAsync(payload.react as React.ReactElement);
```

Comment:
```suggestion
      payload.html = await render(payload.react as React.ReactElement);
```

let's use the non-deprecated `render` method

Reply:
This is the same as the `emails` method. Going to add the same dynamic import though